kitt_bar_app/current_batch.rb

```diff
@@ -111,6 +114,7 @@ def parse_batch_status
   end
 
   def batch_open?
-    @api_data.dig('status') != 500
+    # status is in api_data only when we don't have a 500 or 403
+    !@api_data&.dig('status')
```

Comment:
This will return true if `@api_data` is `nil`, which is weird 😅. Anyway @api_data won't be nil, I'm working on it :)

Reply:
Ok my bad, @pi_data should be nil if batch doesn't exist.

So this is an issue as we display the batch header if it exists with this method.

We then have to replace this by
`@api_data && @api_data.dig('status')`

Reply:
Ok I just got it. 😅
`@api_data && @api_data.dig('status').nil?`

Reply:
But I was also working on displaying potential errors on batches. So this might not be relevant anymore